sandbox/public/functional-test/alloyVisitorTestPageInt.html

```diff
@@ -1,15 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Are these files moving to the repo that @shammowla is creating?

Reply:
I only moved the "perf" directory from "alloy/sandbox/public/".

Reply:
What @shammowla said. The need for `alloyVisitorTestPageInt.html` and its variants goes away entirely due to the other changes in this PR. Now we just have a single test page and we just inject scripts dynamically as needed based on the purmutation needed for the test (all handled in `clientScripts.js`.